docs/source/quickstart.rst

```diff
@@ -193,3 +183,44 @@ which returns::
     {"predictions": [1, 0]}
 
 For more information, see :doc:`models`.
+
+
+Logging to a Remote Tracking Server
+-----------------------------------
+In the examples above, MLflow logs data to the local filesystem of the machine it's running on.
+To manage results centrally or share them across a team, you can configure MLflow to log to a remote
+tracking server. To get access to a remote tracking server:
+
+- :ref:`Launch a tracking server <tracking_server>` on a remote machine.
+
+- Alternatively, sign up for `Databricks Community Edition <https://databricks.com/try-databricks>`_,
+  which comes bundled with a hosted, auto-updating tracking server that lies behind authentication. Note that
```

Comment:
"that lies behind authentication" is kind of awkward. Maybe just say "a hosted tracking server".